oauth2_provider/views/base.py

```diff
@@ -183,6 +185,13 @@ class TokenView(OAuthLibMixin, View):
     @method_decorator(sensitive_post_parameters("password"))
     def post(self, request, *args, **kwargs):
         url, headers, body, status = self.create_token_response(request)
+        if status == 200:
+            access_token = json.loads(body).get('access_token')
+            if access_token is not None:
+                token = AccessToken.objects.get(token=access_token)
```

Comment:
`AccessToken` is a swappable model, you need to instantiate is using `get_access_token_model`